src/photos/ducks/albums/components/AlbumsView.jsx

```diff
@@ -20,19 +21,21 @@ const Content = ({ list }) => {
   }
 }
 
-export default class AlbumsView extends Component {
-  render() {
+class AlbumsView extends Component {
+  render({ t }) {
```

Comment:
This way of getting props being specific to Preact, we shouldn't add more but preferably remove what we may find, should we?